airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -334,6 +355,52 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return f"repos/{stream_slice['repository']}/labels"
 
 
+class Organizations(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/orgs#get-an-organization
+    """
+
+    def __init__(self, organizations: List[str], **kwargs):
+        super(GithubStream, self).__init__(**kwargs)
+        self.organizations = organizations
+
+    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
+        for organization in self.organizations:
+            yield {"organization": organization}
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"orgs/{stream_slice['organization']}"
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        yield response.json()
+
+
+class Repositories(Organizations):
+    """
+    API docs: https://docs.github.com/en/rest/reference/repos#list-organization-repositories
+    """
+
+    fields_to_minimize = ("owner",)
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"orgs/{stream_slice['organization']}/repos"
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        for record in response.json():  # GitHub puts records in an array.
+            yield self.transform(record=record)
+
+
+class Tags(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/repos#list-repository-tags
+    """
+
+    primary_key = None
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"repos/{stream_slice['repository']}/tags"
+
+
 class Teams(GithubStream):
```

Comment:
Since you created `Organizations` class, could you please make it a parent class for `Teams` class.

Reply:
Done